src/Microsoft.DotNet.ProjectModel/Files/ProjectFilesCollection.cs

```diff
@@ -73,11 +74,7 @@ internal void EnsureInitialized()
                 .ExcludeGroup(_preprocessPatternsGroup)
                 .ExcludeGroup(_resourcePatternsGroup);
 
-            _contentPatternsGroup = PatternGroup.Build(_rawProject, _projectDirectory, _projectFilePath, "content", additionalIncluding: contentBuiltIns, additionalExcluding: excludePatterns.Concat(_publishExcludePatterns))
-                .ExcludeGroup(_compilePatternsGroup)
-                .ExcludeGroup(_preprocessPatternsGroup)
-                .ExcludeGroup(_sharedPatternsGroup)
-                .ExcludeGroup(_resourcePatternsGroup);
+            _contentPatternsGroup = PatternGroup.Build(_rawProject, _projectDirectory, _projectFilePath, "content", additionalIncluding: contentBuiltIns, additionalExcluding: _publishExcludePatterns);
```

Comment:
This is actually wrong. We still want to allow publish exclude. There's just no default. This is nice so that you can choose to do more inclusive globbing